apps/playground/src/app/quiz/containers/results/results.component.html

```diff
@@ -0,0 +1,90 @@
+          <div *ngIf="percentage < 60">
+            <div mat-card-image class="try-again"></div>
+            <h3>Try again! </h3>
+          </div>
+
+          <span *ngIf="elapsedMinutes < 1">You scored {{ percentage }}% correctly (and quickly)!</span>
+          <span *ngIf="elapsedMinutes > 1">You scored {{ percentage }}% correct.</span>
+        </div>
+      </section>
+
+      <section class="quizSummary">
+        <details>
+          <summary>View a more detailed summary of your quiz</summary>
+          <div class="allQuestions" *ngFor="let question of allQuestions">
+            <div class="quiz-summary-question">
+              <div class="quiz-summary-field">
+                <span>
+                  <span class="leader">Question #{{ question.questionId }}: </span>{{ question.questionText }}
+                </span>
+              </div>
+              <div class="quiz-summary-field">
+                <span>
+                  <span class="leader">Your Answer: </span>
+                  Option {{ question.selectedOption }} &mdash; {{ question.options[question.selectedOption - 1].optionText }}
+                  <mat-icon class="correct" *ngIf="question.answer === question.selectedOption">done</mat-icon>
+                  <mat-icon class="incorrect" *ngIf="question.answer !== question.selectedOption">clear</mat-icon>
+                  <span *ngIf="question.selectedOption === ''"> (no answer provided)</span>
+                </span>
+              </div>
+              <div class="quiz-summary-field">
+                <span>
+                  <span class="leader">Correct Answer: </span>
+                  Option {{ question.answer }} &mdash; {{ question.options[question.answer - 1].optionText }}
+                </span>
+              </div>
+              <div class="quiz-summary-field">
+                <span>
+                  <span class="leader">Explanation: </span>
+                  Option {{ question.answer }} was correct because {{ question.explanation }}.
+                </span>
+              </div>
+            </div>
+          </div>
+        </details>
+      </section>
+    </section>
+
+    <section class="return">
+      <mat-card-actions>
+        <a routerLink="/quiz/intro" class="btn">Restart Quiz</a>
+        <a href="https://www.codelab.fun" class="btn">Back to Codelab</a>
+      </mat-card-actions>
+    </section>
+
+    <hr />
+
+    <section class="challenge-social">
+      <h4>Challenge your friends!</h4>
+      <div class="social-buttons">
+        <a class="btn email"
+           href="mailto:?subject=Try to beat my quiz score!&amp;body=I scored {{ percentage }}% on this awesome Angular quiz. Try to beat my score at {{ codelabUrl }}">Email a friend</a>
+        <a class="btn twitter" target="_blank"
+           href="https://www.twitter.com/share?text=I scored {{ percentage }}% on this awesome Angular quiz. Try to beat my score at &amp;hashtag=quiz&amp;url={{ codelabUrl }}">Tweet your score</a>
```

Comment:
Cool, didn't know you could do that

Reply:
yes, I wanted to add Facebook but couldn't quite figure it out...